SUPPORTEDTECH.rst

```diff
@@ -0,0 +1,111 @@
+Supported Technology
+====================
+
+Supported Operations
+####################
+
+.. table::
+
+    +--------------+------------------------+-----------------------------------------+-----------------------+
```

Comment:
I would suggest you make it operation-wise. 

Like 

Operation               Protocol 
addition          |       SPDZ,Falcon,FSS 
mul, Matmul,conv2d and conv2d_transpose  |  SDPZ,Falcon,FSS

Reply:
I have done another iteration on that. It looks better now, should I let the number of parties column there or its okay now?

Reply:
Also, I do not know which protocol support which of this operations: (tanh, exponential, relu, ...). I am not able to see in the code how they are related. Are all of them supported by each protocol with all security guarantees?